### ISSUE TITLE
Fix adding locations by geometry

### ISSUE THREAD

**Issue by brianherbert** [ushahidi/crowdmap-issues#319]
Originally reported by @dkobia 

@brianherbert @evansims - this commit broke crowdmap

Fatal error: Cannot use object of type stdClass as array in
/root/cm3/api/helpers/geometry.php on line 194

**Comment by brianherbert** [comment]
@dkobia What data are you using to produce this error?

**Comment by dkobia** [comment]
Just trying to create a new post with a location.

**Comment by dkobia** [comment]
Example Post:

```
{
  "apikey": "XXXXXXXX",
  "locations[lat]": 37.785835266113281,
  "locations[lon]": -122.4064178466797,
  "message": "Test Fix It! #fixit #garbage"
}
```

**Comment by brianherbert** [comment]
Ah, so not using a geometry, just a lat,lon pair.

Thanks!

**Comment by evansims** [comment]
This appears to have been resolved with ushahidi/Crowdmap-New@149aa7b17bdaf968ae4e61f8c4cc5c5b5311a271; closing.
